recipes/cowpy/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "cowpy" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "1bdc61d107df02fd34a9241f2220d0704a01d8ce16bed8bff3512a34a0efa56a"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
```

Comment:
can this be `noarch: python`

Reply:
I don't want it to be noarch because the conda constructor currently doesn't support noarch packages, and I want to distribute this package with a custom Miniconda Installer.

Reply:
Have you tried conda-pack?

Reply:
I hadn't heard of conda-pack! Thanks for the suggestion, I will try it today!

Reply:
conda-pack is not working for me. :cry: